src/databricks/labs/ucx/installer/policy.py

```diff
@@ -0,0 +1,94 @@
+import json
+import logging
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.service import compute
+
+logger = logging.getLogger(__name__)
+
+
+class ClusterPolicyInstaller:
+    def __init__(self, installation: Installation, ws: WorkspaceClient):
+        self._ws = ws
+        self._installation = installation
+        self._policy_id = ""
+
+    @staticmethod
+    def _policy_config(value: str):
+        return {"type": "fixed", "value": value}
+
+    def create_cluster_policy(
```

Comment:
```suggestion
    def create(
```
cluster policy is already in the name of the class. keep it DRY (don't repeat yourself)